src/components/magnet/magnet-canvas.tsx

```diff
@@ -71,9 +72,24 @@ export class MagnetCanvas extends BaseComponent<IProps, IState> {
     return newState;
   }
 
+  private strengthDisposer: IReactionDisposer;
+
   constructor(props: IProps) {
     super(props);
     this.state = {};
+
+  }
+
+  public componentDidMount() {
```

Comment:
We touched on this briefly in Slack, but it would be nice to hear any additional feedback on this approach to picking up MST changes in the Canvas components